src/main/java/org/broadinstitute/barclay/help/BashTabCompletionDoclet.java

```diff
@@ -0,0 +1,356 @@
+package org.broadinstitute.barclay.help;
+
+import com.sun.javadoc.RootDoc;
+import freemarker.template.Configuration;
+import freemarker.template.Template;
+import freemarker.template.TemplateException;
+
+import java.io.*;
+import java.util.*;
+
+/**
+ * For testing of help documentation generation.
+ */
+public class BashTabCompletionDoclet extends HelpDoclet {
+
+    // Barclay BashTabCompletionDoclet custom Command-line Arguments:
```

Comment:
Can you add a comment to each of the options and/or variables saying what they are meant for? I have little experience with bash-completion and it will be useful for me to understand the code better. Maybe this could be added to the class javadoc to allow developers to know the options available and what are they use for...